content/howtos/easy-model-import-gallery.md

```diff
@@ -7,7 +7,7 @@ weight = 2
 
 Now lets pick a model to download and test out. We are going to use `luna-ai-llama2-uncensored.ggmlv3.q5_K_M.bin`, there are a few ways to do this, https://huggingface.co/TheBloke/Luna-AI-Llama2-Uncensored-GGML/blob/main/luna-ai-llama2-uncensored.ggmlv3.q5_K_M.bin
 
-In the `docker` cmd run this. This uses the Gallery to download the model, it may also set up the yaml file, but we will need to override that for the how to setup!
```

Comment:
This will need to be added back in. As there was some confusion on where to run the command.

Reply:
Okay I see what you mean now.  I have added it back in, but expanded upon it to make more sense.

Let me know what you think!